schemas/fws0_fwr_status.fbs

```diff
@@ -0,0 +1,24 @@
+// Schema for Kafka-to-NeXus Streamers status
+
+file_identifier "fws0";
+
+struct Statistics {
+avg : float;
+std : float;
```

Comment:
There's no advantage to using `avg` and `std` abbreviations here. `std` is particularly unclear (standard deviation, standard error?).